### PR TITLE
CI trying to get to Green

### PR DESCRIPTION
**Pull request by bvanessen**
Changed the limit on MACC WAE model to minimize transient errors.  Upped the time limit on the Tioga tests.

### CODE REVIEW

File: python/lbann/launcher/flux.py [hunk 1]
```diff
@@ -5,13 +5,13 @@
...
-    hours, minutes = divmod(int(minutes), 60)
-    days, hours = divmod(hours, 24)
-    return f'{days}-{hours:02}:{minutes:02}:{seconds:02}'
+# def _time_string(minutes):
+#     """Time D-hh:mm:ss format."""
+#     minutes = max(minutes, 0)
+#     seconds = int(round((minutes % 1) * 60))
+#     hours, minutes = divmod(int(minutes), 60)
+#     days, hours = divmod(hours, 24)
+#     return f'{days}-{hours:02}:{minutes:02}:{seconds:02}'
```
Comment (tbennun) [inline python/lbann/launcher/flux.py:14]:
```suggestion
```

File: scripts/build_lbann.sh [hunk 2]
```diff
@@ -48,7 +48,7 @@ CMD_LINE_VARIANTS=
 
 # Default versions of Hydrogen, DiHydrogen, and Aluminum - use head of repo
 HYDROGEN_VER="@develop"
-ALUMINUM_VER="@1.3.1:"
+ALUMINUM_VER="@master"
```
Comment (tbennun) [inline scripts/build_lbann.sh:51]: we may want to release a version here. OK for now

**Review by tbennun — APPROVED** [review]
Looks good with one tiny commented-code-removal change

File: scripts/build_lbann.sh (hunk 2 above)
Comment (ndryden) [reply scripts/build_lbann.sh:51]: Waiting for LLNL/Aluminum#208 to merge before doing another release.

**Review by ndryden — APPROVED** [review]

**Pull request merged**